alibi_detect/cd/pytorch/utils.py

```diff
@@ -1,5 +1,10 @@
+import logging
```

Comment:
Even though the `get_torch_device` utility is currently only used for drift detectors, it would probably be used for outlier/adversarial detectors too as we'll be extending them to the PyTorch backend. Therefore it would make more sense to add `get_torch_device` to `alibi_detect.utils.pytorch` instead. There isn't a perfect "fit" for it yet but `misc.py` could make sense @ascillitoe ? Moreover, it could be fine to just call it `get_device` since it's imported from `alibi_detect.utils.pytorch` anyway. But that's a very tiny detail! :)

Reply:
So `get_device` in `alibi_detect.utils.pytorch`? 🙂

Reply:
I can make this change but i feel we should adopt a principle like YAGNI/KISS here and leave it as is till we are actually there.